defaults/main.yml

```diff
@@ -347,7 +358,9 @@ postgresql_cpu_index_tuple_cost:       0.005   # same scale as above
 postgresql_cpu_operator_cost:          0.0025  # same scale as above
 postgresql_parallel_tuple_cost:        0.1     # same scale as above (>= 9.6)
 postgresql_parallel_setup_cost:        1000.0  # same scale as above (>= 9.6)
-postgresql_min_parallel_relation_size: 8MB     # (>= 9.6)
+postgresql_min_parallel_relation_size: 8MB     # (= 9.6) replaced by below two parameters in 10
```

Comment:
Do we need the comment about "replaced below"?  I'm not sure if that matches the style of the others, which normally just list > < = and the version, etc.

Reply:
It may be helpful as a reminder if transitioning from 9.6 to 10.